submissions/mustbefail/dom-api-task/script.js

```diff
@@ -0,0 +1,62 @@
+const init = () => {
+  const imgBaseUrl = "http://image.tmdb.org/t/p/w300/";
+  const API_KEY = "6a4ca56d29d657ff19ba2cf05591a088";
+
+  const cardsContainer = document.querySelector(".cards-container");
+  const genreList = document.querySelectorAll("[data-genre]");
+  const menuChevron = document.querySelector(".menu-chevron");
+
+  const getMoviesByGenre = async (movieGenre) => {
+    const reqUrl = `https://api.themoviedb.org/3/discover/movie?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1&primary_release_year=2020&with_genres=${movieGenre}`;
+    const response = await fetch(reqUrl);
+    const data = await response.json();
+    render(data);
```

Comment:
If you are working with an API you should [handle network errors](https://www.tjvantoll.com/2015/09/13/fetch-and-errors/)